lightning/src/blinded_path/message.rs

```diff
@@ -74,16 +68,106 @@ impl Writeable for ForwardTlvs {
 	}
 }
 
+/// Similar to [`ForwardTlvs`], but these TLVs are for the final node.
+pub(crate) struct ReceiveTlvs {
+	pub tlvs: Option<RecipientData>
+}
```

Comment:
Let's move this back where it was to keep the blame log clean.

Reply:
Done! [8e4e1aa](https://github.com/lightningdevkit/rust-lightning/commit/8e4e1aabadb5fda9f4d0ec2c845263c94c273643)
Thanks!